### PR TITLE
Displaying pages in footer added

### PR DESCRIPTION
**Pull request by rafalkowalski**
#1562 after merge please add pages in settings

### CODE REVIEW

**Comment by rafalkowalski** [comment]
@niranjan94 please review

File: open_event/templates/gentelella/admin/footer.html [hunk 1]
```diff
@@ -57,17 +58,11 @@ <h4 class="footer-header">Use Open Event</h4>
...
-                    <a href="https://opev.wordpress.com/">Blog</a>
-                    <a href="/support">Help</a>
-                    <a href="/press">Press</a>
-                    <a href="/security">Security</a>
-                    <a href="/developers">Developers</a>
-                    <a href="/privacy">Privacy</a>
-                    <a href="/cookies">Cookies</a>
-                    <a href="/sitemap.xml" class="hide-large">Sitemap</a>
+                    {% for page in system_pages %}
+                        {% if page.place == 'footer_menu'%}
```
Comment (SaptakS) [inline open_event/templates/gentelella/admin/footer.html]: This change makes the UI unresponsive.

**Comment by niranjan94** [comment]
@rafalkowalski fix the build please :)

**Comment by niranjan94** [comment]
@rafalkowalski tests are still failing :sweat_smile:

**Pull request merged**